Makefile

```
@@ -86,6 +86,14 @@ mod-outdated: ## check outdated dependencies
 authors: ## generate AUTHORS file from git history
 	scripts/docs/generate-authors.sh
 
+.PHONY: completion
+completion: binary
+completion: /etc/bash_completion.d/docker
+completion: ## generate and install the completion scripts
+
+/etc/bash_completion.d/docker: FORCE ## generate and install the bash-completion script
```

I think we can just use .PHONY here:

```suggestion
.PHONY: /etc/bash_completion.d/docker
/etc/bash_completion.d/docker:
```

---

Ah, yeah, both should work here; I picked `FORCE` to make it explicit it's not a "fake" target, so an actual file, but would still have to be forced; I had some recollection hat was the preferred way somewhere, but can't find that back (maybe it was some discussion in another project).

---

updated 👍